rpcd/etc/openstack_deploy/user_osa_variables_defaults.yml

```diff
@@ -39,3 +39,8 @@ octavia_git_repo: 'https://git.openstack.org/openstack/octavia'
 octavia_git_project_group: 'octavia_all'
 octavia_git_install_fragments: "venvwithindex=True&ignorerequirements=True"
 
+# RabbitMQ
```

Comment:
Wrong file - this stuff now goes into https://github.com/rcbops/rpc-openstack/blob/newton/group_vars/all/apt.yml with the rest of the apt repo configs.